### DISCUSSION TITLE
How to create a proxy site?

### DISCUSSION THREAD

**Discussion by behitek** [EasyEngine/easyengine#1793] [Support Requests]
I would like to create a proxy site:
- I have a service running on port 3000, and I would like to create a site mapping from the domain to that service.
- Using WordOps, I can create the proxy site like the picture below.
<img width="802" alt="image" src="https://github.com/EasyEngine/easyengine/assets/12376486/1fd73be1-2f62-4e7c-a9e3-672064e2f0b1">

But I can't find the document or something related to easyengine, so I created this discussion for your help.
Thanks.

**Comment by umar13893** [comment]
Anyone know solution for this?
